models.go

```diff
@@ -16,6 +16,7 @@ const (
 	NIP_47_GET_INFO_METHOD            = "get_info"
 	NIP_47_MAKE_INVOICE_METHOD        = "make_invoice"
 	NIP_47_LOOKUP_INVOICE_METHOD      = "lookup_invoice"
+	NIP_47_PAY_KEYSEND_METHOD         = "pay_keysend"
```

Comment:
can you please also add to `NIP_47_CAPABILITIES`

Reply:
Done!